sdk/storage/azure-storage-blobs/test/block_blob_client_test.cpp

```diff
@@ -45,7 +45,7 @@ namespace Azure { namespace Storage { namespace Test {
     m_blobUploadOptions.HttpHeaders.ContentDisposition = "attachment";
     m_blobUploadOptions.HttpHeaders.CacheControl = "no-cache";
     m_blobUploadOptions.HttpHeaders.ContentEncoding = "identity";
-    m_blobUploadOptions.HttpHeaders.ContentMd5 = "";
```

Comment:
Should HttpHeaders be containing `ContentHash` or `ContentMd5`?

Reply:
The type is `std::vector<uint8_t>` so I think it should be `ContentMd5`

Reply:
We agreed on this yesterday

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/1213